app/processing/delta/calculate-line-deltas.js

```diff
@@ -2,10 +2,10 @@ const calculateLineDeltas = (invoiceLines) => {
   return [...invoiceLines.reduce((x, y) => {
     // group by line types, so create key representing the combination
     // exclude account code as past requests vary based on ledger
-    const key = `${y.schemeCode}-${y.fundCode}-${y.description}`
+    const key = `${y.schemeCode}-${y.fundCode}-${y.agreementNumber ?? 'default'}-${y.description}`
```

Comment:
Suggest putting `'default'` in a constant so it's easier to update in future.

Would probably be fine just declared in this file as this is it's only value, but happy to take your judgement on that